admin/Default/changelog.php

```diff
@@ -6,7 +6,7 @@
 $template->assign('ChangeMessage', $var['change_message'] ?? '');
 $template->assign('AffectedDb', $var['affected_db'] ?? '');
 
-$db2 = new SmrMySqlDatabase();
+$db2 = MySqlDatabase::getInstance(true);
```

Comment:
I wonder if adding a `MySqlDatabase::getNewInstance()` may be nice to have so that there is a little more clarity as to what's going on.

Reply:
I have yet to review the new interface, but my gut reaction is that a separate, explicit method would be nice. Alternatively, this type of interface would be a candidate for named arguments, which are coming in PHP8.

Reply:
Yeah but the named arguments might be even more wordy :D I'm also sort of leaking implementation details here, so perhaps concealing those is a better option